cli/tauri.js/api/window.js

```diff
@@ -0,0 +1,14 @@
+import tauri from './tauri'
+
+function setTitle (title) {
+  tauri.setTitle(title)
+}
+
+function open (url) {
+  tauri.open(url)
+}
+
```

Comment:
we should add the ability to change window size, transparency (maybe in the near future!)

Reply:
@lucasfernog - what do you think about this?

Reply:
I'll make it on a second PR. This one is too big already, but I want to make more additions.

Reply:
window size change is only supported by zserge, so i'll do it when we get the chance.